git/foundations_git/commit_messages.md

```diff
@@ -42,7 +42,7 @@ This is the change I made to the codebase.
 
 <div class="lesson-note lesson-note--tip" markdown=1>
 
-GitHub has a 72-character limit so we recommend keeping your commits' subject to within this amount.
+GitHub has a 72-character limit, so we recommend keeping your commit's subject within this amount.
```

Comment:
Since people will be making multiple commits over time, it makes more sense to use a plural possessive with the note talking about commits as a whole. That being said, it would need the plural `subjects` though.

```suggestion
GitHub has a 72-character limit, so we recommend keeping your commits' subjects within this amount.
```

Reply:
In this context your suggestion is also suitable, I just think this part of the lesson regards to creating a good commit message so the singular would be appropriate.

Reply:
If the note was part of a task involving making a commit e.g. as in the Git Basics lesson, then yeah absolutely since it'll be referring to the commit that the user is currently making.

In this scenario, the lesson is about commit messages in general, as is the note, so using plural is appropriate here. But the current grammar is understandably odd since the "subject" bit is incorrectly singular. Happy to merge when that's amended :+1: